README.md

```diff
@@ -285,6 +286,17 @@ existing fidoconfig file if you have one.
    othernets do not (regularly) distribute a nodediff file, but just hatch a
    new nodelist every few weeks.  In this case, you need the FullUpdate
    statement.
+```
+   Dailynodelist
+   ----------
+   Syntax:  Dailynodelist <On/Off>
+   Example: Dailynodelist On
+```
+   This statement tells to nlupdate to process a daily nodelist, you have
```

Comment:
This statement tells to nlupdate to process a daily nodelist, you have => This statement tells nlupdate to process a daily nodelist. You have